nanome/plugin_init.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import os
+import re
+import sys
+import zipfile
+
+TEMPLATE_ZIP = os.path.join(os.path.dirname(__file__), 'plugin-template.zip')
+
+def main():
+    path = sys.argv[1] if len(sys.argv) > 1 else '.'
+    fields = {
+        'name': 'Plugin',
+        'description': 'A Nanome Plugin',
+        'category': 'other',
+        'version': '1.0.0',
```

Comment:
A lot of python package seems to start with 0.1.0 as a first version. Do you prefer 1.0.0?